Makefile.toml

```diff
@@ -281,7 +281,7 @@ cargo test \
 
 # host-ctr unit tests (go)
 ${BUILDSYS_TOOLS_DIR}/docker-go \
-  --module-path ${BUILDSYS_SOURCES_DIR}/host-ctr \
```

Comment:
I know that `ecs-gpu-init` currently doesn't have any unit tests, but we can still add it here in case that it does in the future. Not sure how much overhead this would incur though? Maybe it's not worth it?

Reply:
I test it, and I didn't see any slowdown in the execution so I'll keep it.